BlueprintUI/Sources/Internal/Logger.swift

```diff
@@ -131,6 +131,60 @@ extension Logger {
         )
     }
 
+    static func logCacheHit(object: AnyObject, description: @autoclosure () -> String, constraint: SizeConstraint) {
```

Comment:
Yeah, for sure, I thought about wiring up all the Logger methods to something besides `os_signpost` in order to collect data for analysis outside of Instruments, but didn't get to it. This can get hit a lot of times very frequently during a render pass, so collecting this data in production might be impractical due to the perf hit.